logRootQA.py

```diff
@@ -205,12 +205,12 @@ def summaryJR(jrDir):
             nDiff=nDiff+len(diffs)
         logs=getFiles(root+'/'+d,'*.log')
         nAll+=len(logs)
-        if len(logs)==1:
-            output=runCommand(['grep','DONE calling validate',logs[0]])
+        for log in logs:
+            output=runCommand(['grep','DONE calling validate',log])
             if len(output[0])>0:
                 nOK+=1
             else:
-                print('JR results failed',d)
+                print('JR results failed in ',log)
```

Comment:
this does not function with pr_testing/run-pr-comparisons `grep -a 'JR results failed' $QALOG | awk '{ split($0,a," "); print "<tr><td><a href=\"validateJR/"a[4]"\">"a[4]"</a></td><td align=\"right\"><a href=\"validateJR/"a[4]"/"a[4]".log\">log</a></td></tr>" }' >> $JRHTML`